src/redux/store.js

```diff
@@ -1,10 +1,8 @@
 import { configureStore } from '@reduxjs/toolkit';
-import missionReducer from './mission/missionSlice';
 import rocketReducer from './rocket/rocketSlice';
 
 const Store = configureStore({
   reducer: {
-    mission: missionReducer,
```

Comment:
Hi, @H-Ramalan I wanted to know if you've removed the mission slice from the store. Before we merge, I want us to clarify that before. Thank you